ccp/src/main/res/values-fr/strings.xml

```diff
@@ -244,7 +244,7 @@
     <string name="zambia">Zambie</string>
     <string name="zimbabwe">Zimbabwe</string>
     <string name="unknown">Inconnu</string>
-    <string name="search">Recherche...</string>
+    <string name="search">Recherche…</string>
```

Comment:
You can use \u2026 instead 
https://www.fileformat.info/info/unicode/char/2026/index.htm

Reply:
What's the advantage of that?

Reply:
The first one would be read out as "three period" in talkback
The unicode variant would just skip the talkback here.

Reply:
Interesting! Done 0459676bf6aba57cf2aa0985ab653c360a18b6d3